rules_building_block/execution_script_process.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/07/11"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/11"
+
+[rule]
+author = [ "Elastic" ]
+building_block_type = "default"
+description = "Execution via script process"
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Execution via Script Process"
+risk_score = 21
+rule_id = "8af93dc4-4d00-4b93-944b-e7d8c4bbb2ed"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "OS: macOS", "Use Case: Threat Detection", "Tactic: Execution", "Rule Type: BBR"]
+type = "eql"
+timestamp_override = "event.ingested"
+query = '''
+   process where event.type == "start" and
+   process.name : ("bash", "dash", "sh", "tcsh", "csh", "zsh", "ksh", "fish", "python*",
+   "perl*", "php*")
```

Comment:
This rule will trigger on every single shell command being ran. I think this is too noisy, and I also don't see the benefits of having a rule like this. Maybe just exclude it from BBR conversion?

Reply:
Discussed in RAD removing noisy rules.